packages/connect-query/tsconfig.build.json

```diff
@@ -1,7 +1,11 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "verbatimModuleSyntax": true,
```

Comment:
I think it would be best to enable verbatimModuleSyntax in tsconfig.base.json. This show errors in the IDE instead of later in the build.